wled00/data/settings_leds.htm

```diff
@@ -339,17 +342,25 @@
 						LC.min = 0;
 						LC.style.color="#fff";
 						return; // do not check conflicts
-					}
-					else if (isHub75(t)) {
-							LC.max = 128;
-							LC.min = 16;
-							LC.style.color="#fff";
-							return; // do not check conflicts
 					} else {
 						LC.max = d.max_gpio;
 						LC.min = -1;
 					}
 				}
+				if (isHub75(t) && (nm=="L0" || nm=="L1")) {
+					// Matrix width and height
+					LC.max = 128;
+					LC.min = 16;
```

Comment:
you can add `LC.step=16;` to limit entries to multiples of 16.
Be sure to clear step otherwise.

Reply:
Actual values supported are 16,32,64,128

This an example of where I want the bus config to be able to supply a enum

Reply:
Let's expedite @willmmiles "End oappend" and then this will be easier to implement in JS and backend code.

Reply:
Yeah I thought that would help. I've also been playing with a few ideas as how to reduce repetition but also allow per ledType customisations 

E.g we don't want to have to pass "data pin" for every led and "clock pin" for every one that features clock - we already have the cover with the D and 2P type.

For the exception, like for Hub75 we can supply the optional "overrides" for any labels, input fields active, input types etc

Reply:
We could do something similar as it is done for usermods: `addDD()`.